docs/concepts/select.md

```diff
@@ -353,6 +353,61 @@ export class CityService {
 
 now we can use this `zooThemeParks` selector anywhere in our application.
 
+### Inheriting selectors  
+
+Another way to share selectors across states with similar structure could be to create dynamic selectors in a base class. Using the same previous example we could:
+
+```TS
+export class EntitiesState {
+
+  static entities() {
+    return createSelector([this], (state: { entities: any[] }) => {
```

Comment:
should be it be `state: string[]`? because below states has the `string[]` model :D

Reply:
right, it was after the dynamics selectors section before

Reply:
Maybe we can use generic here?

For example
```export class EntitiesState<T>```